app/views/coronavirus_landing_page/_nhs_logo.html.erb

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 69.2 28" alt="NHS logo">
+    <title>NHS</title>
+    <style>
+        .st0 {
+            fill: #005EB8;
+        }
+        
+        .st1 {
+            fill: #FFFFFF;
+        }
+    </style>
+    <path class="st0" d="M0 0h69.2v28H0z" />
+    <path class="st1" d="M6.8 2.7h7.5l4.6 15.6h.1l3.1-15.6h5.7L23 25.2h-7.4L10.8 9.6h-.1L7.6 25.2H2L6.8 2.7zM30 2.7h6l-1.7 8.6h7.1l1.8-8.6h6l-4.7 22.5h-6l2-9.6h-7.2l-2 9.6h-6L30 2.7zM65.1 7.8c-1.6-.7-3.3-1.1-5-1-2.4 0-4.3.3-4.3 2.2 0 3.2 8.8 2 8.8 8.9 0 6.2-5.8 7.9-11 7.9-2.4-.1-4.7-.4-7-1.2L48 20c1.7.9 3.6 1.3 5.5 1.3s4.8-.4 4.8-2.7c0-3.6-8.8-2.3-8.8-8.6 0-5.8 5.1-7.6 10-7.6 2.3-.1 4.7.2 6.9 1l-1.3 4.4z" />
+    <%= tag.image(src: image_url("nhs-logo.png"),
+        alt: "NHS logo",
+        "xlink:href": "") %>
```

Comment:
This looks important but also potentially easily overlooked. Is this documented in the commit message or somewhere?

Reply:
I've updated the commit message with a reference and explaination